src/couch_replicator_manager.erl

```diff
@@ -182,7 +182,7 @@ init(_) ->
         scan_pid = ScanPid,
         max_retries = retries_value(
             config:get("replicator", "max_replication_retry_count", "10")),
-        rep_start_pids = [Pid],
+        rep_start_pids = [{rep_start, Pid}],
```

Comment:
right idea but `Pid` here isn't a replication that's starting, it's the changes reader.